### PR TITLE
Set up manual build stages for travis

### PR DESCRIPTION
**Pull request by jakemac53**
Working towards https://github.com/dart-lang/angular/issues/820, all manual edits for now

### CODE REVIEW

**Comment by kevmoo** [comment]
Crazy cool, @jakemac53 !

The last master branch build took over 2hrs - https://travis-ci.org/dart-lang/angular/builds/336288174

**Comment by jakemac53** [comment]
@alorenzen @matanlurey are you willing to take this PR as is while I work on mono_repo support? Or would you prefer to wait?

File: angular/lib/source_gen.dart [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /// @nodoc
 library angular.source_gen;
 
+// meaningless change
```
Comment (kevmoo) [inline angular/lib/source_gen.dart]: Delete this before you commit! @jakemac53

Comment (jakemac53) [reply angular/lib/source_gen.dart]: done

**Comment by matanlurey** [comment]
Looks good. Let's merge this internally if you're ready @jakemac53?

**Comment by jakemac53** [comment]
Yep, ready whenever. Not sure exactly what the workflow looks like here, did you want me to do it or do you folks handle it?

**Comment by matanlurey** [comment]
@jakemac53 I'll handle it today!

**Comment by matanlurey** [comment]
Merged. Will sync in the next batch.

**Pull request closed without merging**